src/lib/lang-runner-java.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as https from 'https';
+import * as execa from 'execa';
+import LangRunner from '../lib/lang-runner';
+import LocalRun from '../lib/local-run';
+
+const runtimeJarName = 'sf-fx-runtime-java-runtime-1.0.3-jar-with-dependencies.jar';
+const runtimeJarUrl = `https://repo1.maven.org/maven2/com/salesforce/functions/sf-fx-runtime-java-runtime/1.0.3/${runtimeJarName}`;
+const runtimeJarDir = path.resolve(os.tmpdir(), 'sf-fx-runtime-java-runtime-jar');
+const runtimeJarPath = path.resolve(runtimeJarDir, runtimeJarName);
+const runtimeJarSha = '1db6d78bdbb7aff7ebe011565190ca9dd4d3e68730e206628230d480d057fe1e';
+
+export default class LangRunnerJava extends LangRunner {
```

Comment:
@Malax do you mind reviewing this file? @mimen was less sure about reviewing this since he doesn't have Java context.